src/signal.c

```diff
@@ -25,27 +25,13 @@ void signal_USR1(int, siginfo_t *, void *);
 void signal_SEGV(int, siginfo_t *, void *);
 void signal_BUS(int, siginfo_t *, void *);
 
-struct sigaction saTERM = {.sa_handler = NULL,.sa_sigaction = signal_TERM,
```

Comment:
on many modern systems, `.sa_handler` and `.sa_sigaction` are a `union`, therefore initializing them both is bad.